tools/dispatch_data.py

```diff
@@ -44,16 +44,11 @@ def submit_jobs(args) -> str:
     graph_name = schema_map["graph_name"]
 
     # retrieve num_parts
-    num_chunks = len(schema_map["num_nodes_per_chunk"][0])
-    num_parts = num_chunks
+    num_parts = args.num_parts
     partition_path = os.path.join(args.partitions_dir, "partition_meta.json")
     if os.path.isfile(partition_path):
         part_meta = load_partition_meta(partition_path)
         num_parts = part_meta.num_parts
-    if num_parts > num_chunks:
```

Comment:
we support `num_parts` less than `num_chunks` now?

Reply:
Yes. We support no. of chunks < no. of partitions now.